hooks/environment

```diff
@@ -8,16 +8,22 @@ get-ssm () {
   aws ssm get-parameter --name "$1" --query "Parameter.Value" --with-decryption --output text
 }
 
+add-ssh-key () {
+  if [ -n "${1}" ]
+  then
+    echo "Adding ${1} deploy key to a default SSH agent"
+    # shellcheck disable=SC2046
+    eval $(/usr/bin/ssh-agent | sed "/^echo/d")
+    get-ssm "${1}" | ssh-add - 2>/dev/null
+  fi
+}
+
 # shellcheck disable=SC1090
 [ -f "${_source_dir}/custom-defaults" ] && source "${_source_dir}/custom-defaults"
 
-if [ -n "${AWS_PARAMSTORE_SECRETS_GLOBAL_SSH}" ]
-then
-  echo "Adding Global SSH deploy key to a default SSH agent"
-  # shellcheck disable=SC2046
-  eval $(/usr/bin/ssh-agent | sed "/^echo/d")
-  get-ssm "${AWS_PARAMSTORE_SECRETS_GLOBAL_SSH}" | ssh-add - 2>/dev/null
-fi
+add-ssh-key "${AWS_PARAMSTORE_SECRETS_GLOBAL_SSH}"
+
+add-ssh-key "${AWS_PARAMSTORE_SECRETS_GLOBAL_SSH_ALT}"
```

Comment:
Can we instead of creating a new envvar, re-use the existing var which can optionally be a list of string catenated by colon? I mean the `AWS_PARAMSTORE_SECRETS_GLOBAL_SSH` could be either:

`/apps/buildkite/some-global-ssm-path`

and

`/apps/buildkite/some-global-ssm-path:/apps/buildkite/another-global-ssm-path`.

Where in later case we will add two keys to the agent.

Reply:
@xiaket applied your suggestions